src/Alarm.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import soundfile from './audio/alarm.mp3';
+import Sound from 'react-sound';
+
+class Alarm extends Component {
+    constructor(props) {
+        super(props);
+    }
+    render() {
+        return (
+            this.props.playAlarm && <Sound
+                url={soundfile}
+                playStatus={Sound.status.PLAYING}
+                onLoading={this.handleSongLoading}
```

Comment:
I can not find these functions handleSongLoading, handleSongPlaying and handleSongFinishedPlaying